erts/configure.in

```diff
@@ -3905,6 +3905,7 @@ dnl use "PATH/include" and "PATH/lib".
 AC_SUBST(SSL_INCLUDE)
 AC_SUBST(SSL_INCDIR)
 AC_SUBST(SSL_LIBDIR)
+AC_SUBST(SSL_DEFINE)
```

Comment:
@legoscia We would prefer renaming SSL_DEFINE to SSL_FLAGS  to make it similar to other flags.